readthedocs/templates/docsitalia/overrides/builds/build_detail.html

```diff
@@ -35,7 +35,7 @@
       <div data-bind="visible: finished()">
         <li>
           {% trans "Completed" %}
-          <span>{{ build.date|date:"N j, Y. P" }}</span>
+          <span>{{ build.date|date:"j N Y P" }}</span>
```

Comment:
Userei `"j N Y G:i"` per avere le ore e i minuti nel formato italiano (ad es.: 23:59, `a.m.` e `p.m.` non sono comuni in Italia)

Reply:
Aggiornato!